drivers/SmartThings/deepsmart/profiles/Ac.yml

```diff
@@ -0,0 +1,22 @@
+name: Ac.v1
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: temperatureMeasurement
+    version: 1
+  - id: thermostatHeatingSetpoint
```

Comment:
We are already modifying the test

Reply:
Accessing bridge through HTTPS to obtain devices

Reply:
SSDP searches for all gateways within the local area network. It obtains all devices under the gateway through HTTPS and obtains the properties of all current devices through HTTPS. Then, every 2 seconds, it obtains devices with changed properties in the gateway through HTTPS. If there are any, it obtains all properties of the changed devices through HTTPS. 
Control is also achieved through HTTPS controlling sub devices through the gateway.
Support multiple gateway access.

Reply:
1 discover the bridge and use https api to get devices via the bridge
2 driver uses https api(get all devices/control device/query device) to interact with bridge
3 via bridge over https
4 driver polls the updated devices every 2 seconds, and then query polled devices over https
4 no, 6 apis
 1)  get all devices in the bridge 
 GET https://ip/homecontroller/api/v1/config/devices
 2) get device configs
 GET https://ip/homecontroller/api/v1/config/dp2knx
3) get device enum configGET https://ip/homecontroller/api/v1/config/dpsamenum
4) get updated devices since given time
POST https://ip/homecontroller/api/v1/config/changeddevs{"command":"getchangeddevs","params":{"time":"1712110490"}}
5) query device(addr is knx address like a/b/c)
POST https://ip/homecontroller/api/v1/query
{"command":"query","addrs":[1,2,3]}
6) control device
POST https://ip/homecontroller/api/v1/control
{"command":"control","cmslist":[{"addr":1,"type":0,"value":1,"delay":0}]}

6 bridge
   dp2knx is device config
   devices in bridge are all confided by TSL(things specification language)
   pid1 is one kind of ac
  pid2 is another kind of ac
   but in St pid1&&pid2 are same so we make the dp2knx to figure out how different ACs connected to st
  dpenum is properties config. for example, ac has several  modes(hot wind cold dry auto wet…), pid1 may have the value 0,1,2,3,4,5…
but pid2 may have the value 1,3,5,7,9,11…
we use dpenum to make different acs can be controlled by the same st mode value